model-optimizer/extensions/front/tf/space_to_batch.py

```diff
@@ -60,9 +63,8 @@ def find_and_replace_pattern(self, graph: Graph):
 
             diff_size = Sub(graph, {'name': node.name + '/sub_0'}).create_node()
             diff = Sub(graph, {'name': node.name + '/sub_1'}).create_node()
-
             const_begin = Const(graph, {'value': int64_array([1])}).create_node()
-            const_pad_val = Const(graph, {'value': int64_array([1])}).create_node()
+            const_pad_val = Const(graph, {'value': int64_array(1)}).create_node()
```

Comment:
Implement a unit-test for this change.

Reply:
I was surprised to see no transformation test for this replacement. So, I added an assert in the `Pad` layer to fail if it faces non-scalar 3rd input -- it is totally compliant with specification so the assertion is valid. And it fail model TF models conversion with BTS or STB if we wont perform this change. However, I filed a ticket to cover this transformation with transformation test: 34514. Is that OK?

Reply:
Yes, looks good.